python/graphstorm/dataloading/sampler.py

```diff
@@ -210,3 +210,43 @@ def _gen_neg_pair(pos_pair, utype, vtype):
             pos_neg_tuple = _gen_neg_pair(pos_pairs,
                 g.canonical_etypes[0][0], g.canonical_etypes[0][2])
         return pos_neg_tuple
+
+class JointLocalUniform(JointUniform):
+    '''Jointly corrupt a group of edges.
+    The main idea is to sample a set of nodes and use them to corrupt all edges in a mini-batch.
+    This algorithm won't change the sampling probability for each individual edge, but can
+    significantly reduce the number of nodes in a mini-batch.
+
```

Comment:
It's hard to understand from the docstring how it is different than JointUniform or other sampler. Can you please clarify a little more?

Reply:
Done